pages/management/index.vue

```diff
@@ -130,7 +132,8 @@ export default Vue.extend({
         {
           hid: 'description',
           name: 'description',
-          content: 'Login with your favorite wallet provider to manage your project or proposal.',
+          content:
+            'Login with your favorite wallet provider to manage your project or proposal.',
```

Comment:
nitpick(blocking): use a const or even better a i18n variable. I also requested this in another pr, just skip it here if you already did this.

Reply:
Created an ticket for this to align this up for the entire project. See #458